packages/overture-schema-core/src/overture/schema/core/ref.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from enum import Enum
+
+from overture.schema.core.models import Feature
+
+
+class Relationship(Enum):
+    """
+    Category of relationship between a value that refers to another value.
+
+    If we call the first value, the one that holds the reference, the referent; and the second
+    value, the one that is referred to, as the referee; then this value represents the relationship
+    from the perspective of the referent.
+    """
+
+    def __init__(self, value: str, doc: str) -> None:
+        self._value_ = value
+        self.__doc__ = doc
+
+    BELONGS_TO = "belongs_to", "The referent belongs to the referee"
+    BOUNDS = "bounds", "The referent bounds the referee"
+    CONNECTED_TO = "is_connected_to", "The referent is connected to the referee"
```

Comment:
Has one / has many? Or is this spatial / node / vertex-based?

Reply:
It's topologically based I guess. Any thoughts on a better name and/or description?

Reply:
`CONNECTS_TO` is parallel to `BELONGS_TO`.

Seth found that in multiple places - RDF, OWL.

Reply:
Just updated the PR with a commit that applies this change.